rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -2827,28 +2835,22 @@ void main_window::dropEvent(QDropEvent* event)
 	}
 	case drop_type::drop_rap: // import rap files to exdata dir
 	{
-		int installed_rap_count = 0;
-
 		for (const auto& rap : drop_paths)
 		{
 			const std::string rapname = sstr(QFileInfo(rap).fileName());
 
 			if (InstallRapFile(rap, rapname))
 			{
 				gui_log.success("Successfully copied rap file by drop: %s", rapname);
-				installed_rap_count++;
 			}
 			else
 			{
 				gui_log.error("Could not copy rap file by drop: %s", rapname);
 			}
 		}
 
-		if (installed_rap_count > 0)
-		{
-			// Refresh game list since we probably unlocked some games now.
-			m_game_list_frame->Refresh(true);
-		}
+		// Refresh game list since we probably unlocked some games now.
+		m_game_list_frame->Refresh(true);
 		break;
```

Comment:
Why did you change this?

Reply:
Figured this is unnecessary. Worth noting that I'm the one who added this bit all the way back in 2018.

Reply:
A: this is my code
B: this is not bloat

You should actually add this logic to the other code section instead of removing it here

Reply:
I'm not trying to step on anyone's toes, I was just implying that I didn't pick some random snippet to change.